tests/containers/rootless_podman.pm

```diff
@@ -36,14 +36,14 @@ sub run {
 
     install_podman_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
-    my $user      = $testapi::username;
-    my $check_msg = 'Checking allocation range of user';
-    script_run "grep $user /etc/subuid || echo /etc/subuid has no uid range for $user", output => $check_msg;
-    script_run "grep $user /etc/subgid || echo /etc/subgid has no gid range for $user", output => $check_msg;
-    assert_script_run "usermod --add-subuids 200000-201000 --add-subgids 200000-201000 $user";
+    my $user         = $testapi::username;
+    my $subuid_start = get_user_subuid($user);
+    if ($subuid_start eq '') {
+        $subuid_start = 200000;
+        assert_script_run "usermod --add-subuids $subuid_start-201000 --add-subgids $subuid_start-201000 $user";
```

Comment:
```suggestion
        assert_script_run "usermod --add-subuids 200000-201000 --add-subgids 200000-201000 $user";
```

Reply:
The end of the range is hardcoded, to the start should be hardcoded as well. If you change the assignment one line up, you'd end up with wrong ranges.

Reply:
you are right. would it make sense to calculate the second part then? ex: $subuid_start+1000?